lib/Command/UpsertProvider.php

```diff
@@ -176,10 +193,60 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($mapping = $input->getOption('mapping-uid')) {
 			$this->providerService->setSetting($provider->getId(), ProviderService::SETTING_MAPPING_UID, $mapping);
 		}
+		if ($mapping = $input->getOption('mapping-fediverse')) {
+			$this->providerService->setSetting($provider->getId(), ProviderService::SETTING_MAPPING_FEDIVERSE, $mapping);
+		}
+		if ($mapping = $input->getOption('mapping-groups')) {
+			$this->providerService->setSetting($provider->getId(), ProviderService::SETTING_MAPPING_GROUPS, $mapping);
+		}
```

Comment:
You forgot to add this option in the configure method. This will make the integration tests pass.